src/pymodaq/examples/qt_less_standalone_module.py

```diff
@@ -73,13 +90,23 @@ def send_data(self, grabber_type: str = "") -> None:
         print("send_data")
 
     # actuator commands
-    def move_abs(self, position: Union[float, str]) -> None:
-        print("move_abs", position)
-        self._fake_position = float(position)
-
-    def move_rel(self, position: Union[float, str]) -> None:
-        print("move_rel", position)
-        self._fake_position += float(position)
+    def move_abs(
```

Comment:
why not a position as a DataActuator here and only a float? Some actuators are having their "position" as a vector or even an "image" (for instance for my spatial light modulator)

Reply:
That `position` argument is only able to receive JSON serializable content.
PyMoDAQ objects are transferred binary in the `additional_payload` argument.